redaxo/src/addons/structure/pages/index.php

```diff
@@ -206,7 +206,12 @@
         $i_category_id = $KAT->getValue('id');
 
         $kat_link = $context->getUrl(['category_id' => $i_category_id]);
-        $kat_icon_td = '<td class="rex-table-icon"><a href="' . $kat_link . '" title="' . htmlspecialchars($KAT->getValue('catname')) . '"><i class="rex-icon rex-icon-category"></i></a></td>';
+        $kat_icon_td = '<td class="rex-table-icon"><a href="' . $kat_link . '" title="' . htmlspecialchars($KAT->getValue('catname')) . '"><i class="fa fa-folder-o"></i></a></td>';
+		$kat_icon_td_has_children = '<td class="rex-table-icon"><a href="' . $kat_link . '" title="' . htmlspecialchars($KAT->getValue('catname')) . '"><i class="fa fa-folder"></i></a></td>';
+		
+		if(rex_category::get($KAT->getValue('id'))->getChildren()) {
+						$kat_icon_td = $kat_icon_td_has_children;
+					}
```

Comment:
Nicht das gesamte HTML doppelt schreiben. Es ändert sich ja nur das Icon. Das könntest du auch in der if/else-Kurzschreibweise notieren. Auch würde ich das ausgefüllte Icon anzeigen, wenn die Kategorien normale Artikel enthält. Aktuell würde hier das Outline-Icon kommen.

Bitte auch keine `fa-` direkt notieren. Wie verwenden `rex-icon` und dann die Icon-Class. So kann jedes Theme seinen eigenen Iconfont mitbringen. Für das outline Icon würde ich die Class `rex-icon-category-without-elements` vorschlagen und müsste mit in die `_icon.scss` aufgenommen werden.